WalletWasabi.Tests/UnitTests/BranchAndBoundSelectionTests.cs

```diff
@@ -0,0 +1,105 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.Blockchain.TransactionOutputs;
+using WalletWasabi.BranchNBound;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.BranchAndBoundTests
+{
+	public class BranchAndBoundSelectionTests
+	{
+		private static Random Random = new Random();
+		private static List<Money> AvailableCoins = GenList();
+
+		private static List<Money> GenList()
+		{
+			List<Money> availableCoins = new List<Money>();
+			for (int i = 0; i < 1000; i++)
+			{
+				availableCoins.Add((ulong)Random.Next((int)Money.Satoshis(1000), (int)Money.Satoshis(99999999)));
```

Comment:
No need to cast it to `ulong`.
```suggestion
				availableCoins.Add(Random.Next((int)Money.Satoshis(1000), (int)Money.Satoshis(99999999)));
```

Reply:
True, I only casted to avoid `CS0618` warning.